### PR TITLE
Update progress messages

### PR DESCRIPTION
**Pull request by bigfluffycookie**
fixes https://github.com/SonarSource/sonarlint-visualstudio/issues/4421

### CODE REVIEW

File: src/ConnectedMode/Migration/ConnectedModeMigration.cs [hunk 1]
```diff
@@ -130,14 +130,16 @@ private async Task MigrateImplAsync(BoundSonarQubeProject oldBinding, IProgress<
...
                 logger.WriteLine(MigrationStrings.Process_SkippingChecking);
             }
 
-            progress?.Report(new MigrationProgress(0, 1, "Create new binding files ...", false));
+            progress?.Report(new MigrationProgress(0, 1, "Creating new binding files ...", false));
             logger.WriteLine(MigrationStrings.Process_ProcessingNewBinding);
 
             var progressAdapter = new FixedStepsProgressToMigrationProgressAdapter(progress);
             await unintrusiveBindingController.BindAsync(oldBinding, progressAdapter, token);
+            progress?.Report(new MigrationProgress(0, 1, "Saving new binding files ...", false));
```
Comment (bigfluffycookie) [inline src/ConnectedMode/Migration/ConnectedModeMigration.cs:142]: To me it feels weird not having this here so I added it but lmk what you think

File: src/ConnectedMode/Migration/ConnectedModeMigration.cs [hunk 2]
```diff
@@ -109,7 +109,7 @@ private async Task MigrateImplAsync(BoundSonarQubeProject oldBinding, IProgress<
             var legacySettings = await settingsProvider.GetAsync(oldBinding.ProjectKey);
 
             // TODO: add proper progress messages.
-            progress?.Report(new MigrationProgress(0, 1, "Getting files ...", false));
+            progress?.Report(new MigrationProgress(0, 1, "Getting files to clean...", false));
```
Comment (duncanp-sonar) [inline src/ConnectedMode/Migration/ConnectedModeMigration.cs:112]: Maybe `Finding files to clean ...`?

The other messages have a space before the `...` - it looks a bit on the screen, I think.

File: src/ConnectedMode/Migration/ConnectedModeMigration.cs (hunk 1 above)
Comment (duncanp-sonar) [reply src/ConnectedMode/Migration/ConnectedModeMigration.cs:142]: I don't think it's needed, but it doesn't make a big difference.
Does it cause the vertical scroll bar to appear? If so, it means the users will always have to scroll to see the last message, which isn't ideal.

File: src/ConnectedMode/Migration/ConnectedModeMigration.cs [hunk 3]
```diff
@@ -146,7 +148,7 @@ private async Task MigrateImplAsync(BoundSonarQubeProject oldBinding, IProgress<
             logger.WriteLine(MigrationStrings.Process_DeletingSonarLintFolder);
             await fileSystem.DeleteFolderAsync(legacySettings.LegacySonarLintFolderPath);
 
-            progress?.Report(new MigrationProgress(0, 1, "Finished", false));
+            progress?.Report(new MigrationProgress(0, 1, "Finished!", false));
```
Comment (duncanp-sonar) [inline src/ConnectedMode/Migration/ConnectedModeMigration.cs:151]: `Migration finished successfully` ? (with or without the !)

FYI the other possible final messages are:
* `Migration cancelled` and 
* `Error during migration. See the Output Window for more information`

**Review by duncanp-sonar — APPROVED** [review]
Looks good. A couple of minor suggestions.

**Pull request merged**